Sources/MapboxCoreNavigation/BundleAdditions.swift

```diff
@@ -113,25 +113,21 @@ extension Bundle {
      Returns the value associated with the specific key in the Mapbox Navigation bundle's information property list, if installed.
      */
     public class func string(forMapboxNavigationInfoDictionaryKey key: String) -> String? {
-        if let stringForKey = Bundle.mapboxNavigationIfInstalled?.object(forInfoDictionaryKey: key) {
-            return stringForKey as? String
-        } else if let infoDictionary = Bundle.mapboxNavigationInfoDictionary {
+        if let infoDictionary = Bundle.mapboxNavigationInfoDictionary {
             return infoDictionary[key] as? String
         } else {
-            return nil
+            return Bundle.mapboxNavigationIfInstalled?.object(forInfoDictionaryKey: key) as? String
```

Comment:
Since this is a public method, let’s update the changelog to mention that we fixed an issue where it returned values from the application bundle instead of the navigation SDK when statically linking using CocoaPods.